ui/app/AppLayouts/Profile/views/MyProfileView.qml

```diff
@@ -112,41 +114,34 @@ SettingsContentBase {
     readonly property var priv: QtObject {
         id: priv
 
-        property bool hasAnyProfileShowcaseChanges: false
+        property bool hasAnyProfileShowcaseChanges: showcaseModels.dirty
+
+        property ProfileShowcaseModels showcaseModels: ProfileShowcaseModels {
+            communitiesSourceModel: root.communitiesModel
+            communitiesShowcaseModel: root.profileStore.profileShowcaseCommunitiesModel
+            
+            accountsSourceModel: root.walletStore.accounts
+            accountsShowcaseModel: root.profileStore.profileShowcaseAccountsModel
+
+            collectiblesSourceModel: root.profileStore.collectiblesModel
+            collectiblesShowcaseModel: root.profileStore.profileShowcaseCollectiblesModel
+        }
 
         function reset() {
             descriptionPanel.displayName.text = Qt.binding(() => { return profileStore.displayName })
             descriptionPanel.bio.text = Qt.binding(() => { return profileStore.bio })
             profileStore.resetSocialLinks()
             profileHeader.icon = Qt.binding(() => { return profileStore.profileLargeImage })
 
-            profileShowcaseCommunitiesPanel.reset()
-            profileShowcaseAccountsPanel.reset()
-            profileShowcaseCollectiblesPanel.reset()
-            profileShowcaseAssetsPanel.reset()
+            priv.showcaseModels.revert()
```

Comment:
`d` instead of `priv`?

Reply:
Nope, it's a trap! SettingsContentsBase already has a `d` component, so this will silently fail :)